packages/provwasm-test-tube/src/module/msgfees.rs

```diff
@@ -23,15 +26,31 @@ where
         pub assess_custom_msg_fee: MsgAssessCustomMsgFeeRequest["/provenance.msgfees.v1.MsgAssessCustomMsgFeeRequest"] => MsgAssessCustomMsgFeeResponse
     }
 
-    fn_query! {
-        pub query_params ["/provenance.msgfees.v1.Query/Params"]: QueryParamsRequest => QueryParamsResponse
+    fn_execute! {
```

Comment:
none of these tx are something that a person/contract can execute directly ... they would need a gov proposal to change these settings within the msg fee module.

Reply:
The queries as well?

Reply:
Queries are fine... it is just that the msg fee settings are governance controlled.  A user might want to know what fees are configured though so the queries make sense.

Reply:
fixed